packages/react-components/react-card/e2e/Card.e2e.tsx

```diff
@@ -240,4 +241,50 @@ describe('Card', () => {
       });
     });
   });
+
+  describe('selection', () => {
+    it('should not be selectable by default', () => {
+      mountFluent(<CardSample />);
+
+      cy.get(`.${checkboxClassNames.input}`).should('not.exist');
+    });
+
+    it('should show checkbox when selectable', () => {
+      mountFluent(<CardSample selectable />);
+
+      cy.get(`.${checkboxClassNames.input}`).should('exist').should('not.be.checked');
+    });
+
+    it('should select with a mouse click', () => {
+      mountFluent(<CardSample selectable />);
+
+      cy.get(`.${checkboxClassNames.input}`).realClick();
+
+      cy.get(`.${checkboxClassNames.input}`).should('be.checked');
+    });
+
+    it('should select with the Enter key', () => {
```

Comment:
Correct, it's one of the defined interactions:

### Keyboard interaction
1. Focus Initial focus is set on card, to access items within the card use EnterKey or Spacebar
2. Focus-Inside Focus moves to first actionable item in the card. Use ArrowKeys or TabKey to move focus to child elements within the card. To exit card use EscKey, to move focus back to card [parent]
3. Checkbox-Focus Depending on your tab order of items within the card, the number of keystrokes will vary. Once focus is set on the checkbox by default it will be set to unselected
4. SpaceBar or EnterKey will toggle both Card and Checkbox to selected state. Keying again will toggle to unselected

I'll add a Spacebar behavior too.